Unix/configeditor/configeditor.cpp

```diff
@@ -336,16 +349,22 @@ int MI_MAIN_CALL main(int argc, const char** argv)
         size_t pos = line.find("=");
         if (line.empty() || pos == string::npos)
         {
-            cout << line << std::endl;
+        	if (opts.query.empty())
+        	{
+                cout << line << std::endl;
```

Comment:
I think we should keep the output no matter the query is specified or not specified.